wpimath/src/main/java/edu/wpi/first/math/geometry/Translation2d.java

```diff
@@ -36,6 +36,13 @@
 @JsonAutoDetect(getterVisibility = JsonAutoDetect.Visibility.NONE)
 public class Translation2d
     implements Interpolatable<Translation2d>, ProtobufSerializable, StructSerializable {
+  /**
+   * A preallocated Translation2d representing the origin.
+   *
+   * <p>This exists to avoid allocations for common translations.
+   */
+  public static final Translation2d kZero = new Translation2d();
```

Comment:
Late to the party, but would it make team code more readable if we called this `kOrigin` instead?

Reply:
It might be useful to add an alias `kOrigin = kZero` for `Translation2d` and `Translation3d`, but we should keep `kZero` for consistency with the other classes. (Origin is specific to positions/translations, but all groups must have a zero element)

Reply:
Why is kOrigin clearer than kZero?